Civilla-Project/Police-Control/src/main/java/BotActions.java

```diff
@@ -0,0 +1,186 @@
+import java.util.ArrayList;
+
+interface BotAction {
```

Comment:
Maybe change the name of the interface to IBotAction for compliance with naming style?

Reply:
Nice point. Will do it.